homeassistant/components/connector/__init__.py

```diff
@@ -0,0 +1,109 @@
+"""The Connector integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+from socket import timeout
+
+from connectorlocal.connectorlocal import WIFIMOTORTYPE, ConnectorHub
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_HOST, EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import (
+    DOMAIN,
+    KEY_COORDINATOR,
+    KEY_GATEWAY,
+    KEY_MULTICAST_LISTENER,
+    MANUFACTURER,
+)
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = ["cover"]
```

Comment:
```suggestion
PLATFORMS: list[Platform] = [Platform.cover]
```
You should use the Platform constants from homeassistant.const